spacy/ml/tb_framework.pyx

```diff
@@ -561,3 +559,15 @@ cdef void _sum_state_features(CBlas cblas, float* output,
                 feature = &cached[idx]
             saxpy(cblas)(O, one, <const float*>feature, 1, &output[b*O], 1)
         token_ids += F
+
+cdef inline int _arg_max(const float* scores, const int n_classes) nogil:
```

Comment:
Probably not for this PR, but we have other argmax implementations in `spacy/ml/parser_model.pyx`. It would be nice if they were somewhere together.

Reply:
`spacy/ml/parser_model.pyx` is no longer part of the parser codebase post-refactor, right? I can't find it in this branch anymore.

Reply:
Ah, right, I was looking at the wrong branch. There is another one in `spacy/pipeline/_parser_internals/transition_system.pyx`.